src/quickpicks/utils/schemas.test.ts

```diff
@@ -149,23 +148,7 @@ describe("quickpicks/utils/schemas.ts promptForSchema()", () => {
   });
 
   it("should throw an error when no schema versions are found for the subject", async () => {
-    getSchemasForEnvironmentIdStub.resolves([]);
-
-    await assert.rejects(
-      async () => promptForSchema(TEST_LOCAL_KAFKA_TOPIC, "value", SubjectNameStrategy.TOPIC_NAME),
-      { message: `No schema versions found for subject "${TEST_LOCAL_SCHEMA.subject}".` },
-    );
-    assert.ok(showErrorNotificationStub.calledOnce);
-  });
-
-  it("should throw an error when schema versions exist but none match the subject", async () => {
```

Comment:
This test is not redundant with the prior one, so get rid of. The result of `getSchemaSubjectGroup()` is inherently single-subject-related.